project-api/project-api-test/src/test/java/io/ballerina/projects/test/TestBuildProjectWithGeneratedSources.java

```diff
@@ -105,6 +105,37 @@ public void testBuildProjectWithNoReadPermission() {
         resetPermissions(generatedDirPath);
     }
 
+    @Test (description = "tests loading a valid build project with a new generated module")
+    public void testProjectWithNewGeneratedModule() {
+        Path projectPath = RESOURCE_DIRECTORY.resolve("new_generated_mod");
```

Comment:
"new_generated_mod" project does not seem to be committed.

Reply:
Added with https://github.com/ballerina-platform/ballerina-lang/pull/39150/commits/90f975deaa66543c8cbe1727b91888035bb97c1a